wg-advisors/README.md

```diff
@@ -91,6 +91,13 @@ the indulgence of the PMC, advisor. Do not abuse this relationship.
 
 ## FAQ
 
+## How to become an Advisor?
+
+Just subscribe to dev@community.apache.org and (TODO: how) add yourself
```

Comment:
> how

As stated below that "An Advisor must be an ASF member", I think that add oneself to the list can be done with a lazy consensus.

Or if we're confident that the advisor WG is active enough, 3 +1 from the WG.

I personally trust a member and their "describe your experience, skill and interests. Introduce yourself as a prospect Advisor" would show enough motivation.

Reply:
I agree, I think it should be enough to have a member to open a PR with this information here - we will keep a list/table of advisors with information. 

It's a public pleace - not guarded by membership access (which is good) so no email addresses just maybe name, github id, apache id, and short paragraph about the Adviso should be enough,

Reply:
Clarified and updated all these points in the new version.